### ISSUE TITLE
Double wordpress installation

### ISSUE THREAD

**Issue by oskari** [urre/wordpress-nginx-docker-compose-image#6]
Since the primary use for this image is to be used with https://github.com/urre/wordpress-nginx-docker-compose why is there a double wordpress installation? In  this dockerfile wordpress will be installed to /usr/src/wordpress while in the [compose repo composer is used to install wordpress too](https://github.com/urre/wordpress-nginx-docker-compose/blob/master/src/composer.json#L39)

**Comment by urre** [comment]
You're right — bundling WordPress core here was redundant and unused. The consuming project (wordpress-nginx-docker-compose) installs WordPress via Composer/Bedrock into `web/wp` and bind-mounts the app over `/var/www/html`, so `/usr/src/wordpress` was never used.

I've **removed the WordPress core payload** from this image in #8 (which also bumps PHP to 8.3). Closing as addressed there.
